app/models/name.rb

```diff
@@ -439,7 +439,14 @@ def real_search_name
   def self.display_to_real_text(name)
```

Comment:
1.Move to StringExtensions?
2. Is there an elegant, transparent way to do remove and restore group_abbr using only regexp?